apps/docs/content/components/concepts/color-schemes.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Color Schemes (WIP)
+description: This page describes how color schemes works in Hopper, including how applications adapt to operating system dark mode.
+order: 5
+---
+
+## Introduction
+
+Hopper supports light and dark mode. These correspond to color schemes found in popular operating systems. By default, Hopper defaults to the light color scheme, but the `colorScheme` property on the `HopperProvider` can be used to set the color scheme to `dark` or `system` (which will follow the user's operating system setting).
+
+We recommend that all Hopper applications support both light and dark mode. It can be very jarring to use a light themed application when the rest of your applications are in dark mode, and visa versa. This can be accomplished by using Hopper color variables rather than hard coded color values. All Hopper components adapt to color scheme out of the box. See the [styling page](./styling.mdx) for details on how to use Hopper color variables in your own custom components.
+
+## Apply a color scheme
+
+A color scheme can either be enforced by providing a specific light or dark value to a [HopperProvider](../application/hopper-provider.mdx).
+
+{/* TODO: example */}
+```tsx
+    <HopperProvider colorScheme="dark">
+        <Button variant="secondary">Button</Button>
+    </HopperProvider>
+```
+
+A color scheme can also be set to `system`, which will follow the [user's operating system setting](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).
+When the `system` value is provided, an additional fallback color scheme must be specified to defaultColorScheme in case the theme provider is not able to access the user setting.
+
+{/* TODO: example */}
+```tsx
+    <HopperProvider colorScheme="system" defaultColorScheme="light">
+        <Button variant="secondary">Button</Button>
+    </HopperProvider>
+```
+
+Color schemes can also be nested. You could have a dark themed dialog inside a light themed application, for example. This might be useful for certain experiences like color scheme previews, where you want to showcase a specific color scheme, regardless of the operating system setting.
```

Comment:
```suggestion
Color schemes can also be nested.  For example, You could have a dark themed dialog inside a light themed application. This might be useful for certain experiences like color scheme previews, where you want to showcase a specific color scheme, regardless of the operating system setting.
```

I put the for example at the beginning of the sentence since it didn't read well.
If you want it at the end, I actually think there would be no comma.